apps/datahub/src/app/record/navigation-bar/navigation-bar.component.html

```diff
@@ -1,20 +1,39 @@
 <nav
-  class="mx-4 bg-primary flex justify-between py-2 px-4 text-sm rounded-lg text-white sm:space-x-8 sm:px-20 sm:py-5"
+  class="h-full bg-primary-lightest flex justify-between text-sm text-black text-center overflow-hidden shadow-navbar"
+  [class]="
+    displayMobileMenu
+      ? 'items-start'
+      : 'items-center divide-x-2 divide-gray-300'
+  "
 >
+  <div
+    class="h-[60px] flex min-w-[60px] items-center justify-center lg:min-w-40 lg:justify-end lg:pr-4"
+    [class]="displayMobileMenu ? 'border-r-2 border-gray-300' : ''"
+  >
+    <gn-ui-navigation-button (click)="back()" [icon]="'matArrowBack'">
```

Comment:
can't we try using a generic gn-ui-button here? the concept of a "navigation button" is a bit awkward IMO, it's just a button right? plus icons should be properly centered in gn-ui-buttons

Reply:
Right, I'll try to get rid of it.

Reply:
The component has now been removed.